curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcf750a62784cf11f5630.md

```diff
@@ -93,6 +73,19 @@ assert.deepEqual(recipes, [recipe1, recipe2, recipe3]);
 
 # --seed--
 
+## --before-user-code--
```

Comment:
it looks like `--before-user-code--` is being removed [here](https://github.com/freeCodeCamp/freeCodeCamp/issues/57107) 

can you try to see if the `beforeAll` [hook](https://github.com/freeCodeCamp/freeCodeCamp/pull/59001) will work here instead?

Reply:
@jdwilkin4 So is `--before-all--` not still "only supported in challenges that render HTML" as [the docs](https://contribute.freecodecamp.org/how-to-work-on-coding-challenges/) say?

I wanted to ask about this in another issue but forgot.

Reply:
Sorry, I missed the point about logging. Anyways, I agree that the hint should be enough here.